src/debug/di/DI.props

```diff
@@ -77,6 +77,7 @@
     <CppCompile Include="@(SourcesShim)" />
     <CppCompile Include="@(SourcesRightside)" />
     <AssembleAmd64 Condition="'$(BuildArchitecture)' == 'amd64'" Include="..\amd64\floatconversion.asm" />
+    <AssembleArm64 Condition="'$(BuildArchitecture)' == 'arm64'" Include="..\arm64\floatconversion.asm" />
```

Comment:
I believe the way you have it should work. BuildArchitecture gets changed to amd64 for arm64 cross compile case. Is this not working?